# defines bounding box in the frame of the camera from which
# samples will be taken for goal end effector poses
workspace_min: [ 0.5, -0.4,  -0.1 ]
workspace_max: [ 1.0,  0.05, 0.2 ]

sample_resolution_x: 0.15
sample_resolution_y: 0.15
sample_resolution_z: 0.1

roll_offset_degs: 10
pitch_offset_degs: 10
yaw_offset_degs: 10

camera_frame: "asus_rgb_frame"

tracked_marker_id: 8

egress_positions: [
  [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ]
]
